src/thermo/ThermoFactory.cpp

```diff
@@ -129,10 +129,32 @@ ThermoPhase* newPhase(const std::string& infile, std::string id)
 
     if (extension == "yml" || extension == "yaml") {
         AnyMap root = AnyMap::fromYamlFile(infile);
-        AnyMap& phase = root["phases"].getMapWhere("name", id);
-        unique_ptr<ThermoPhase> t(newThermoPhase(phase["thermo"].asString()));
-        setupPhase(*t, phase, root);
-        return t.release();
+        std::string name = id;
+        try {
+            AnyMap& phase = root["phases"].getMapWhere("name", name);
+            unique_ptr<ThermoPhase> t(newThermoPhase(phase["thermo"].asString()));
+            setupPhase(*t, phase, root);
+            return t.release();
+        } catch (CanteraError& err) {
+            if (name=="gri30_mix" || name=="gri30_multi") {
+                warn_deprecated("newPhase",
+                    "\nThe phases named 'gri30_mix' and 'gri30_multi' are no "
```

Comment:
I noticed this when fixing up some of the output for #722... It seems less than optimal that messages have to start with a newline character. I'm not sure how many of these messages are intended to be stretched over multiple lines, but I assume it is a bunch of them. Can this be resolved in `warn_deprecated`? Maybe outside the scope of this PR, but I wanted to mention it (I wish I had seen it in #741)

Reply:
Let's keep this separate, but I'm ok with that change.

Reply:
PS: the intent was to be consistent with the Python `warnings.warn` interface, which does not introduce a newline.

Reply:
In that case, then I don't think you should introduce a newline as the first character here.